### PR TITLE
View comment list

### PR DESCRIPTION
**Pull request by dave-hickman**

### CODE REVIEW

File: src/components/Comments.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    if (isLoading === true){
+        return <p>Loading...</p>
+    }
+    else return (
+      <section>
+        {comments.map((comment) => {
+          return (
+            <article>
+              <p>{comment.author}</p>
+              <p>{comment.created_at}</p>
```
Comment (JVal-NC) [inline src/components/Comments.jsx]: Can we format the date to something more user friendly?

File: src/components/Comments.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            setComments(response.comments)
+            setIsLoading(false)
+        }
+        fetchComments()
+    }, [])
+
+    if (isLoading === true){
+        return <p>Loading...</p>
+    }
+    else return (
```
Comment (JVal-NC) [inline src/components/Comments.jsx:28]: If there are no comments, how does the user know? Would be good if we could have something rendered to the user to idnicate this is the case.

**Review by JVal-NC — COMMENTED** [review]
Dave, really good work so far. Just some minor comments for you to consider in the review.

**Review by xViolaine — APPROVED** [review]
Looking good, nice implementation of the changes suggested!

**Pull request merged**